vizro-core/docs/pages/user_guides/navigation.md

```diff
@@ -149,8 +181,169 @@ If you want to deviate from the default title `SELECT PAGE` and instead provide
+<td style="width:360px">
+
+    ```py
+    vm.Navigation(
+        selector=vm.Accordion(
+            pages={"Accordion title": [
+                "My first page",
+                "My second page"
+            ]}
+        )
+    )
+    ```
+
+</td>
+</tr>
+</table>
+
+### NavBar
+To add navigation bar with icons to your dashboard provide [`NavBar`][vizro.models.NavBar] model to the `selector` argument of the `Navigation`. 
+
+NavBar model has two arguments `pages` and `items`. 
+
+- **`NavBar.pages`**
+
+Use `pages` for default icon image and configuration. 
+   ```
+    Navigation(
+        selector=NavBar(pages=["My first page", "My second page"])
+    )
+   ```
+
+- **`NavBar.items`**
+
+Use `items` to add customization to the icons. `NavBar.items` is provided as list of `NavItem` models. `NavItem` model is responsible for building individual icons within navigation bar. 
+
+For further customizations of icons, you can refer to the [`NavItem`][vizro.models.NavItem] reference. Some popular choices are:
+
+- customize icon image by providing name of the icon from the [Google Material Icon library](https://fonts.google.com/icons) to `icon` argument of `NavItem` model
+- provide text string to be displayed in the icon tooltip on hover by using `text` argument of `NavItem` model
+
+```
+Navigation(
+    selector=NavBar(
+        items=[
+            vm.NavItem(
+                pages=["My first page"], icon="dashboard", text="Pages"
+            ),
+            vm.NavItem(
+                pages=["My second page"], icon="summarize", text="Summary"
+            ),
+        ]
+    )
+)
+```
+
+!!! tip
+    If you would like to turn off icon images you can specify the empty string `""` in the `icon` argument.
+
+#### NavBar examples
+
+- Navigation icons
+<table>
+<tr>
+<td><h3 style="width:350px" align="center"><img align="center" height="450" width="350" src="../../../assets/user_guides/navigation/table_nav1.png"></h3></td>
+<td style="width:360px">
+
+    ```py
+    
+    vm.Navigation(
+        selector=vm.NavBar(
+            pages={
+                "First icon": [
+                    "My first page", 
+                    "My second page"
+                ],
+                "Second icon": [
+                    "My third page"
+                ]
+            }
+        )
+    )
+    
+    ```
+
+</td>
+</tr>
+</table>
+
+- Customizing navigation icons
+
+<table>
+<tr>
+<td><h3 style="width:350px" align="center"><img align="center" height="450" width="350" src="../../../assets/user_guides/navigation/tb_nav_1.png"></h3></td>
+<td style="width:360px">
+
+    ```py
+    
+    vm.Navigation(
+        selector=vm.NavBar(
+            items=[
+                vm.NavItem(
+                    pages=[
+                        "My first page",
+                        "My second page"
+                    ],
+                    icon="dashboard"),
+                vm.NavItem(
+                    pages=["My third page"],
```

Comment:
In this example, when I navigate to the "summarize" icon, the accordion with "My third page" doesn't show up. Is this expected?

![Screenshot 2023-11-10 at 1 31 35 PM](https://github.com/mckinsey/vizro/assets/31973683/40b9ce7c-5f80-4953-b89e-703fa184f4e3)

Reply:
Yes, this is expected. So whenever there is only one page in a accordion, the accordion/nav_panel will not render. And since there are no controls defined on the "My third page" the whole left side is not rendered.